Package@swift-4.swift

```diff
@@ -16,6 +16,7 @@ let package = Package(
   products: [
     .executable(name: "protoc-gen-swift", targets: ["protoc-gen-swift"]),
     .library(name: "SwiftProtobuf", type: .static, targets: ["SwiftProtobuf"]),
+    .library(name: "SwiftProtobufPluginLibrary", targets: ["PluginLibrary"]),
```

Comment:
Does it still need "SwiftProtobuf" on the front of the name since it is scoped to this package?

Reply:
In SwiftPM products share a global namespace (your package declares a dependency on one or more packages, and then your package's targets declare dependencies on targets and products), so the prefix is indeed required. Come to think of it Swift modules live in a global namespace as well, so it might be advisable to rename PluginLibrary to SwiftProtobufPluginLibrary. I'll update the PR with that change.

Reply:
Kevin:  If you could just rename the directory, that seems like the best answer.  (It will require fixing a few other references to get everything to compile cleanly again, of course.)

Thanks for helping with this kind of housekeeping!

Reply:
No problem, updated the PR.